src/components/MainMenu.js

```diff
@@ -17,15 +17,7 @@ const MainMenu = function (props) {
         inverted
         secondary
         size='large'>
-        <Menu.Item>
-          <a
-            href="http://www.codeforboston.org"
-            target="_blank"
-            rel="noopener noreferrer"><Image
-              src={ logo }
```

Comment:
You can get rid of the `logo` and `Image` import statements in this file (lines 5 and 10) now that they're not being used here. I believe that's what's currently causing the Travis build to fail.

Reply:
Thanks! Forgot about those.